bolt-sidecar/bin/sidecar.rs

```diff
@@ -55,3 +57,27 @@ async fn main() -> Result<()> {
         }
     }
 }
+
+fn init_tracing() -> eyre::Result<()> {
+    let std_layer = FmtLayer::default().with_writer(std::io::stdout).with_filter(
+        EnvFilter::builder()
+            .with_default_directive("bolt_sidecar=info".parse()?)
+            .from_env_lossy()
+            .add_directive("reqwest=error".parse()?)
+            .add_directive("alloy_transport_http=error".parse()?),
+    );
+    Registry::default().with(std_layer).try_init()?;
+    Ok(())
+}
```

Comment:
wait, I dont' get this, why are we initializing tracing twice? This is already done inside `init_telemetry_stack`